frontend/src/metabase/ui/components/icons/Icon/Icon.tsx

```diff
@@ -10,6 +10,7 @@ import type { IconName } from "./icons";
 import { Icons } from "./icons";
 
 const defaultSize = 16;
+const multiline = true;
```

Comment:
Should this be a prop?

Reply:
We've talked with @alxnddr and I understood that we want the Tooltip to be multiline by default, everywhere. If needed further on we can indeed make this a prop I suppose?